src/server.rs

```diff
@@ -120,12 +111,12 @@ pub extern "C" fn rustls_server_config_builder_new_with_safe_defaults(
 /// `versions` will only be used during the call and the application retains
 /// ownership. `len` is the number of consecutive `ui16` pointed to by `versions`.
 #[no_mangle]
-pub extern "C" fn rustls_server_config_builder_new(
+pub extern "C" fn rustls_server_config_builder_new_fancy(
```

Comment:
Same comment as on `rustls_client_config_builder_new_fancy`: I think you could provide functions that let clients set cipher suites and TLS protocol version on any `rustls_server_config` and get rid of this entry point.